src/NHSD.BuyingCatalogue.Organisations.Api/Settings/SmtpAuthenticationSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NHSD.BuyingCatalogue.Organisations.Api.Settings
+{
+    /// <summary>
+    /// SMTP authentication settings.
+    /// </summary>
+    internal sealed class SmtpAuthenticationSettings
```

Comment:
Could this class be made immutable? It will make it clearer what properties are required if any.

Reply:
Similar to `EmailMessage` I can't make this one immutable due to its use in configuration – the properties need to be mutable for deserialization.

The required properties depend on the value of `IsRequired`, i.e. `UserName` and `Password` are only required when `IsRequired == true`.

Hopefully this one should be fairly self-explanatory (in terms of what is required when). However, I will make a change so that the SMTP authentication settings can be omitted entirely when authentication is not required.